cli/utils/utils.go

```diff
@@ -61,25 +59,38 @@ var (
 	PrivKeyPassword string
 	Port            uint64
 	OperatorID      uint64
-	DBPath          string
-	DBReporting     bool
-	DBGCInterval    string
 )
 
 // SetViperConfig reads a yaml config file if provided
 func SetViperConfig(cmd *cobra.Command) error {
 	if err := viper.BindPFlag("configPath", cmd.PersistentFlags().Lookup("configPath")); err != nil {
 		return err
 	}
+	if err := viper.BindPFlag("configFileName", cmd.PersistentFlags().Lookup("configFileName")); err != nil {
+		return err
+	}
 	ConfigPath = viper.GetString("configPath")
+	configFileName := viper.GetString("configFileName")
```

Comment:
I meant this could be something simpler like that
```suggestion
	configFileName := viper.GetString("configFileName")
	if configFileName == "" {
	   configFileName = "config.yaml"
	   if cmd.Use != "start-operator" {
	       configFileName = fmt.Sprintf("%s.yaml", cmd.Use)
	   }
	}
	configPathYAML = fmt.Sprintf("%s/%s", ConfigPath, configFileName)
```

Reply:
Yeah, as @RaekwonIII  mentioned at https://github.com/bloxapp/ssv-dkg/pull/37 it turned out not a good idea to have fixed files to look for at `config` folder. I changed this approach back to explicitly providing path to a file `--configPath config.yaml`